src/components/shared/cellRenderers/timeRenderer/timeRenderer.js

```diff
@@ -16,6 +18,12 @@ const formatTime = (value) => {
 export const TimeRenderer = ({ value }) => {
   const formattedTime = formatTime(value);
   return (
-    formattedTime ? formattedTime : EMPTY_FIELD_VAL
+    <div className="pcs-renderer-cell">
```

Comment:
If we want to style error text in a same way, don't we need to wrap empty field value as well?
Before, it was all string values and now TimeRenderer returns div tag while empty value is still remains as string at below.

lastTriggerRenderer.js line 14, 
 ? value.error ? EMPTY_FIELD_VAL : <TimeRenderer value={value.response} /> 

ruleDetails.js line 348, Config.emptyValue

Reply:
The point is to add ellipsis when needed. The error case will just be showing "--" so the ability to do ellipsis there isn't really necessary.